centreon/packages/ui/src/Graph/BarStack/ResponsiveBarStack.tsx

```diff
@@ -0,0 +1,255 @@
+                              )}
+                          </g>
+                        );
+                      })
+                    )
+                  }
+                </BarStackVertical>
+              ) : (
+                <BarStackHorizontal
+                  color={colorScale}
+                  data={[input]}
+                  keys={keys}
+                  xScale={xScale}
+                  y={() => undefined}
+                  yScale={yScale}
+                >
+                  {(barStacks) =>
+                    barStacks.map((barStack) =>
+                      barStack.bars.map((bar) => (
+                        <g key={`bar-stack-${barStack.index}-${bar.index}`}>
+                          <rect
+                            fill={bar.color}
+                            height={bar.height}
+                            key={`barstack-horizontal-${barStack.index}-${bar.index}`}
+                            ry={5}
+                            width={bar.width - 1}
+                            x={bar.x}
+                            y={bar.y}
+                            onClick={() => {
+                              onSingleBarClick?.(bar);
+                            }}
+                            onMouseLeave={() => {
+                              setTimeout(() => {
+                                hideTooltip();
+                              }, 300);
+                            }}
+                            onMouseMove={(event) => {
+                              const eventSvgCoords = localPoint(event);
+                              const top = bar.y + bar.height;
+                              showTooltip({
+                                tooltipData: {
+                                  color: bar.color,
+                                  label: bar.key,
+                                  value: barStack.bars[0].bar.data[barStack.key]
+                                },
+                                tooltipLeft: eventSvgCoords?.x,
+                                tooltipTop: top
+                              });
+                            }}
+                          />
+                          {displayValues &&
+                            bar.height > 10 &&
+                            bar.width > 10 && (
+                              <Text
+                                cursor="pointer"
+                                fill="#000"
+                                fontSize={12}
+                                textAnchor="middle"
+                                verticalAnchor="middle"
+                                x={bar.x + bar.width / 2}
+                                y={bar.y + bar.height / 2}
+                              >
+                                {getValueByUnit({
+                                  total,
+                                  unit,
+                                  value: barStack.bars[0].bar.data[barStack.key]
+                                })}
```

Comment:
Use the `formatMetricValueWithUnit` function from the line chart. It located in src/Graph/common/timeSeries/index.ts

Reply:
I think they are different!

Reply:
But we don't care